hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/HddsDispatcher.java

```diff
@@ -682,6 +697,24 @@ private void audit(AuditAction action, EventType eventType,
     }
   }
 
+  private void performanceAudit(AuditAction action, Map<String, String> params,
+      PerformanceStringBuilder performance) {
+    AuditMessage msg =
+        buildAuditMessageForPerformance(action, params, performance.build());
```

Comment:
Also related to HDDS-9822:

```suggestion
        buildAuditMessageForPerformance(action, params, performance);
```

Reply:
Fixed.